libcxx/docs/TestingLibcxx.rst

```diff
@@ -180,6 +180,42 @@ The tests of libc++ are stored in libc++'s testing related subdirectories:
   ``libcxx/test/libcxx``. The structure of this directories follows the
   structure of ``libcxx/test/std``.
 
+Principles of testing
```

Comment:
I think I might move this before the "structure of the testing related directories" since it's more general?

Reply:
Done.